### PR TITLE
Df error kill processes

### PR DESCRIPTION
**Pull request by dflanderka**
- solve #1048

### CODE REVIEW

File: src/main.cc [hunk 1]
```diff
@@ -74,8 +74,8 @@ it::Record & Application::get_input_type() {
 
 
 
-Application::Application( int argc,  char ** argv)
-: ApplicationBase(argc, argv),
+Application::Application( char ** argv)
```
Comment (jbrezmorf) [inline src/main.cc]: No need for argv.

**Comment by jbrezmorf** [comment]
Remove argv.
Otherwise good, can merge.

File: src/main.cc (hunk 1 above)
Comment (dflanderka) [reply src/main.cc]: We can't remove it, argv[0] is used as argument of PythonLoader::initialize method.

Comment (jbrezmorf) [reply src/main.cc]: OK so just pass argv[0], that's the program path.

**Pull request merged**